understanding/21/content-on-hover-or-focus.html

```diff
@@ -7,6 +7,17 @@
 	</head>
 	<body>
 		<h1>Understanding Content on Hover or Focus</h1>
+
+		<section id="brief">
+			<h2>In brief</h2>
+			<dl>
+				<dt>Objective</dt><dd>More users can perceive and dismiss non-persistent content</dd>
+				<dt>Author task</dt><dd>If content can appear or disappear without a user intentionally activating it, ensure it can persist until dismissed by the user</dd>
```

Comment:
"If content can appear or disappear without a user intentionally activating it..." This phrasing might imply that this SC also covers status messages and "toast" style notifications. 

Perhaps a slight rewording would make it less broad, something like, "If related content is revealed without a user intentionally activating it..."

Reply:
Never going to cover all this in a short description. I've fully revamped to tighten
```suggestion
				<dt>Author task</dt><dd>If hover or focus causes content changes, ensure interaction is predictable</dd>
```

Reply:
+1 for the new wording. Clearly communicates the scope.